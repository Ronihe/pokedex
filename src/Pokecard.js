import React from 'react';

class Pokecard extends React.Component {
  render() {
    return (
      <div class="card">
        <div> {this.props.name}</div>
        <img src={this.props.image} alt="no img found" />
        <div>{this.props.type}</div>
      </div>
    );
  }
}

export default Pokecard;
