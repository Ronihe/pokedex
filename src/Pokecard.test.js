import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Pokecard from './Pokecard';

it('renders without crashing', function() {
  shallow(<Pokecard />);
});

it('matches snapshot for adopted dogs', function() {
  let wrapper = shallow(
    <Pokecard
      name="Charmander"
      type="fire"
      image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/4.png"
    />
  );
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
