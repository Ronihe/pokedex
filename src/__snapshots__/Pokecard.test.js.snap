// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for adopted dogs 1`] = `
<div
  class="card"
>
  <div>
     
    Charmander
  </div>
  <img
    alt="no img found"
    src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/4.png"
  />
  <div>
    fire
  </div>
</div>
`;
